content/en/docs/reference/glossary/quantity.md

```diff
@@ -7,18 +7,19 @@ short_description: >
   A whole-number representation of small or large numbers using SI suffixes.
 
 aka: 
-tags:
+tags: 
+- core-object
```

Comment:
I don't see quantity currently in the [glossary](https://kubernetes.io/docs/reference/glossary/?all=true). I'm guessing it's due to the missing tag here.

Reply:
That's right. I originally thought the file was missing in general, but discovered that the tags field was empty. I guessed on the tag field. In addition, could add fundamental?